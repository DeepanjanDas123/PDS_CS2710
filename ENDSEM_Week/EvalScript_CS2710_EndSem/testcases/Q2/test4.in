DecodeMessage
((g,(j,(l,((t,5),u)))),(9,0));
01001111000111000010011111000010011011011100010011101011001111011011010100111100000110110111001101110001100100111011011011100
Quit
