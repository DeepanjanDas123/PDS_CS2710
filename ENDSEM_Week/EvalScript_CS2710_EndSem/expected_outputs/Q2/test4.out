jugtg9u9gjl0tj5lul0j9uggl0t0tlj590t
